.gitignore

```diff
@@ -75,3 +75,7 @@ cli/build/**
 dbml-error.log
 
 *.profraw
+
+__pycache__
+
+*bin
```

Comment:
What is generating this? I wonder if it can be narrowed down a bit.

Reply:
If its a single folder then perhaps `/bin` or something of that nature would be more adequate. That other `.bin` file could be ignored just in that directory (e.g. `/tools/*.bin` or something like that)